app/server/models/transfer_account.py

```diff
@@ -197,30 +197,28 @@ def get_approval(self, receiving_address):
     def approve_and_disburse(self, initial_disbursement=None, auto_resolve=False):
         from server.utils.access_control import AccessControl
 
-        auto_approve = current_app.config['AUTO_APPROVE_TRANSFER_ACCOUNTS']
         admin = getattr(g, 'user', None)
+        auto_resolve = initial_disbursement != current_app.config['DEFAULT_INITIAL_DISBURSEMENT']
 
-        if not self.is_approved and admin:
-            if AccessControl.has_sufficient_tier(admin.roles, 'ADMIN', 'superadmin'):
-                auto_resolve = True
-                self.is_approved = True
-
-            elif AccessControl.has_sufficient_tier(admin.roles, 'ADMIN', 'admin'):
-                auto_resolve = initial_disbursement \
-                               <= current_app.config['MAXIMUM_CUSTOM_INITIAL_DISBURSEMENT']
-                self.is_approved = auto_approve
-
-            elif AccessControl.has_sufficient_tier(admin.roles, 'ADMIN', 'subadmin'):
-                auto_resolve = False
-                self.is_approved = False
+        if not not self.is_approved and admin and AccessControl.has_sufficient_tier(admin.roles, 'ADMIN', 'admin'):
```

Comment:
shouldn't this be just `if not`? surprised a test didn't catch this given it should probably render approving impossible